### PR TITLE
pass-otp: add page

### PR DESCRIPTION
**Pull request by quentin-bettoum**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pass-otp.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# pass-otp
+
+> A pass extension for managing one-time-password (OTP) tokens.
+> More information: <https://github.com/tadfisher/pass-otp#readme>.
+
+- Prompt for an otpauth URI token and CREATE a new pass file:
```
Comment (einverne) [inline pages/common/pass-otp.md:6]: The intent you use `CREATE` here is clear, but we usually do not use upper case. I prefer just use the word `create`.

What you do think? @sbrl @zdroid

Comment (zdroid) [reply pages/common/pass-otp.md:6]: I agree, good suggestion

Comment (quentin-bettoum) [reply pages/common/pass-otp.md:6]: Ok, I will edit APPEND too.

File: pages/common/pass-otp.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# pass-otp
+
+> A pass extension for managing one-time-password (OTP) tokens.
+> More information: <https://github.com/tadfisher/pass-otp#readme>.
+
+- Prompt for an otpauth URI token and create a new pass file:
+
+`pass otp insert {{pass_path}}`
```
Comment (einverne) [inline pages/common/pass-otp.md:8]:
```suggestion
`pass otp insert {{path/to/pass}}`
```

File: pages/common/pass-otp.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Promt for an otpauth URI token and append to an existing pass file:
+
+`pass otp append {{pass_path}}`
```
Comment (einverne) [inline pages/common/pass-otp.md:12]:
```suggestion
`pass otp append {{path/to/pass}}`
```

File: pages/common/pass-otp.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Print a 2FA code using the OTP token in a pass file:
+
+`pass otp {{pass_path}}`
```
Comment (einverne) [inline pages/common/pass-otp.md:16]:
```suggestion
`pass otp {{path/to/pass}}`
```

File: pages/common/pass-otp.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Copy and don't print a 2FA code using the OTP token in a pass file:
+
+`pass otp --clip {{pass_path}}`
```
Comment (einverne) [inline pages/common/pass-otp.md:20]:
```suggestion
`pass otp --clip {{path/to/pass}}`
```

File: pages/common/pass-otp.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Display a QR code using the OTP token stored in a pass file:
+
+`pass otp uri --qrcode {{pass_path}}`
```
Comment (einverne) [inline pages/common/pass-otp.md:24]:
```suggestion
`pass otp uri --qrcode {{path/to/pass}}`
```

File: pages/common/pass-otp.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Promt for an OTP secret value specifying issuer and account (at least one must be specified) and append to existing pass file:
+
+`pass otp append --secret --issuer {{issuer_name}} --account {{account_name}} {{pass_path}}`
```
Comment (einverne) [inline pages/common/pass-otp.md:28]:
```suggestion
`pass otp append --secret --issuer {{issuer_name}} --account {{account_name}} {{path/to/pass}}`
```

**Comment by quentin-bettoum** [comment]
@einverne I've done it in a [new commit](https://github.com/tldr-pages/tldr/pull/4232/commits/ac718b92349ee7fbb5acbca47999bd8f8bca1d7f) because I wanted to avoid multiple commits for this change (I'm not yet familiar with Github pull requests features). I thought it would automatically validate your requested changes, but apparently no. :slightly_smiling_face:

**Comment by einverne** [comment]
> @einverne I've done it in a [new commit](https://github.com/tldr-pages/tldr/pull/4232/commits/ac718b92349ee7fbb5acbca47999bd8f8bca1d7f) because I wanted to avoid multiple commits for this change (I'm not yet familiar with Github pull requests features). I thought it would automatically validate your requested changes, but apparently no. 🙂

Don't worry about the multiple commits, we will do a squash commit when everything is ok.

**Pull request merged**